teams.yaml

```diff
@@ -5,23 +5,44 @@ teams:
     BRK:
         name: Brockenhurst College
         rookie: No
-    CLY:
-        name: The College of Richard Collyer
+    CLY1:
+        name: The College of Richard Collyer (Team 1)
+        rookie: No
+    CLY2:
+        name: The College of Richard Collyer (Team 2)
         rookie: No
     DCG:
         name: Dr Challoner's High School
         rookie: No
     ELC:
         name: Eltham College
         rookie: No
-    HAB:
-        name: Haberdashers' Aske's Boys' School
+    HAB1:
+        name: Haberdashers' Aske's Boys' School (Team 1)
+        rookie: No
+    HAB2:
+        name: Haberdashers' Aske's Boys' School (Team 2)
+        rookie: No
+    HRS1:
+        name: HIlls Road SIxth Form College (Team 1)
```

Comment:
Is the casing here intentional?

Reply:
I don't think so. It's all copy/pasted from the sheet.

Reply:
Fixed here and in the sheets.

Reply:
👍 Thanks. Looks like `SIxth` is still odd

Reply:
And again: 2af70ce; :facepalm: (also fixed the sheets)